cedar-lean/Cedar/Spec/Ext/IPAddr.lean

```diff
@@ -264,6 +291,15 @@ instance : LT IPNet where
 instance IPNet.decLt (d₁ d₂ : IPNet) : Decidable (d₁ < d₂) :=
 if h : IPNet.lt d₁ d₂ then isTrue h else isFalse h
 
+theorem test1 : unParse ((parse "192.168.0.1/32").get!) = "192.168.0.1/32" := by decide
```

Comment:
I suggest moving these to `UnitTest.IPAddr`: https://github.com/cedar-policy/cedar-spec/blob/main/cedar-lean/UnitTest/IPAddr.lean

Reply:
Moved (and the Decimal tests)